Bottom: 710fd4841e48bbaf2cfbe54a84e1e39df125d31c
Top:    e408dc550942ef1f7036745cee12d162998872e2
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-15 02:03:30 +0000

Refresh of drop-in-pcb-gl-code-various-me

---

diff --git a/src/draw.c b/src/draw.c
index 4605ae5..66ec017 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -518,7 +518,7 @@ DrawEverything (BoxTypePtr drawn_area)
 	r_search (PCB->Data->element_tree, drawn_area, NULL, EMark_callback,
 		  NULL);
       /* Draw rat lines on top */
-      if (PCB->RatOn)
+      if (PCB->RatOn && gui->set_layer ("rats", SL (RATS, 0), 0))
 	DrawRats(drawn_area);
     }
